nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core/Run/RunWorker.cs

```diff
@@ -145,59 +146,51 @@ async Task TrackOriginalContentsAsync(string directory, string fileName)
         }
 
         // do update
-        _logger.Info($"Running update in directory {repoDirectory}");
-        foreach (var project in discoveryResult.Projects)
+        var updateOperations = GetUpdateOperations(discoveryResult).ToArray();
```

Comment:
Most of the changes in this file are whitespace only so I recommend reviewing it that way.